README certificate creation (#132)
elasticpath/react-pwa-reference-storefront: README.md

>         - Replace the `$DOMAIN` parameter in `nginx.conf` with the domain name (excluding the `http://` part, eg. `www.domain.com`) where you plan to host the site.

Reviewer: can you refer https://github.com/elasticpath/react-pwa-reference-storefront/pull/131/files?short_path=d0a94c7#diff-d0a94c79b84b081325ed51136925e08a and rewrite the section ?

Author: Thanks for calling this out. There's actually some incorrect statements in the tutorial which I'll clean up as well now.

        - Replace the `$SSL_CERT_PATH` in `nginx.conf` and `docker-compose.yaml` with the path of the certificate file on the remove server (eg. `/etc/letsencrypt/live/reference.elasticpath.com/fullchain.pem`).
        - Replace the `$SSL_KEY_PATH` in `nginx.conf` and `docker-compose.yaml` with the path of the private key on the remote server (eg. `/etc/letsencrypt/live/reference.elasticpath.com/privkey.pem`).

>         **Note:** You may use a certificate generated through any certificate provider of your choosing. An example is [Let's Encrypt](https://letsencrypt.org).

Reviewer: ```suggestion
        **Note:** You can use the certificate generated by any certificate provider, such as [Let's Encrypt](https://letsencrypt.org).
```
